creme/tree/_attribute_observer/attribute_observer.py

```diff
@@ -29,7 +29,7 @@ def update(self, att_val, class_val, weight):
         class_val: int
             The class value.
 
-        weight: float
+        sample_weight: float
```

Comment:
Use type annotations instead of docstring notes :)

Reply:
I missed that the first time. Now it is fixed :)